### ISSUE TITLE
Build failure (x86_64)

### ISSUE THREAD

**Issue by KevinAdams05** [jackburton79/bescreencapture#33]
BeScreenCapture failes to build on x86_64, hrev52077.

`bin2txt.cpp:17:37: error: no match for 'operator>' (operand types are 'std::basic_istream<char>' and 'int')
   while (std::getline(inFile, line) > 0) {
          ~~~~~~~~~~~~~~~~~~~~~~~~~~~^~~
bin2txt.cpp:17:37: note: candidate: operator>(int, int) <built-in>
bin2txt.cpp:17:37: note:   no known conversion for argument 1 from 'std::basic_istream<char>' to 'int'
`

**Comment by jackburton79** [comment]
Might be caused by newest GCC. Will try to update my x86_64 image and fix
the issue as soon ad possible.

Il dom 8 lug 2018, 04:35 Kevin Adams <notifications@github.com> ha scritto:

**Comment by jackburton79** [comment]
Fixed in master and in the v2.3-release branch.

**Comment by KevinAdams05** [comment]
Thanks!
